.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ethereum Magicians
+    url: https://ethereum-magicians.org/
+    about: Use the Fellowship of Ethereum Magicians for EIP-related discussion!
+  - name: Issue with EIP Bot
+    url: https://github.com/ethereum/EIP-Bot/issues
+    about: If you have a problem with the EIP Bot, report it here!
+  - name: Issue with EIP Validation
+    url: https://github.com/ethereum/eipv/issues
+    about: If you have a problem with EIP Validation, report it here!
+  - name: EIP Improvements
+    url: https://github.com/ethereum-cat-herders/EIPIP/issues
+    about: If you have a suggestion related to the EIP process, submit it here!
```

Comment:
I think the current plan is to use EIP issues for this exact thing, so this one should probably direct to a new issue.